mhkit/wave/resource/environmental_contours.m

```diff
@@ -68,9 +89,16 @@
     dt
     period
     method
-    options.PCA = py.None;
-    options.bin_size = 250;
+    options.min_bin_count = 40;
+    options.initial_bin_max_val = 1;
+    options.bin_val_size = 0.25;
     options.nb_steps = 1000;
+    options.bandwidth = py.None;
+    options.Ndata_bivariate_KDE = 100;
+    options.max_x1 = py.None;
+    options.max_x2 = py.None;
+    options.PCA = py.None;
+    options.PCA_bin_size = 250;
```

Comment:
I think this is a valid strategy to set these values but we will need to ensure that these don't diverge from the default values set in MHKiT-Python